### PR TITLE
Kyle base70

### PR DESCRIPTION
**Pull request by emeraldarcher**
Fixed the Point and Maritime Trade Controllers

### CODE REVIEW

File: java/src/client/maritime/MaritimeTradeController.java [hunk 1]
```diff
@@ -20,11 +20,9 @@
 	public MaritimeTradeController(IMaritimeTradeView tradeView, IMaritimeTradeOverlay tradeOverlay) {
 		super(tradeView);
 		setTradeOverlay(tradeOverlay);
+        tradeView.enableMaritimeTrade(false);
 
 		this.facade = Facade.getInstance();
```
Comment (yeyyosing) [inline java/src/client/maritime/MaritimeTradeController.java]: Do you add the maritime trade controller to the list of observers?

**Comment by emeraldarcher** [comment]
I do now :/

File: java/src/client/services/UserCookie.java [hunk 2]
```diff
@@ -98,6 +98,15 @@ public int getPlayerIndex() {
         return -1;
     }
 
+    public int getVictoryPoints(){
```
Comment (dannyharding10) [inline java/src/client/services/UserCookie.java]: This should go through the facade. Instead of UserCookie

**Comment by yeyyosing** [comment]
Kyle, danny is testing this pull request, and it throws a null exception

File: java/src/client/maritime/MaritimeTradeController.java (hunk 1 above)
Comment (emeraldarcher) [inline java/src/client/maritime/MaritimeTradeController.java]: Now I do...

**Comment by emeraldarcher** [comment]
That's because I need a function that Corbin said he would add to the facade.

**Pull request merged**